### PR TITLE
Init room

### PR DESCRIPTION
**Pull request by trinhtan**

### CODE REVIEW

**Comment by trinhtan** [comment]
image

**Comment by trinhtan** [comment]
ready

**Comment by sokhimeang** [comment]
OK

File: app/views/rooms/rooms_list.html.erb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+    <div class="wrap">
+      <button class="button"><% t ".delete_button" %></button>
+    </div>
+  </form>
+</section>
```
Comment (thanhhungchu95) [inline app/views/rooms/rooms_list.html.erb]: thêm dòng trống cuối file

**Comment by thanhhungchu95** [comment]
commented

**Comment by trinhtan** [comment]
image

**Comment by trinhtan** [comment]
ready

**Pull request merged**